.travis.yml

```diff
@@ -0,0 +1,26 @@
+dist: trusty
+sudo: false
+services:
+  - postgresql
+addons:
+  postgresql: "9.6"
+language: python
+python:
+  - "3.4"
+  - "3.5"
+  - "3.6"
+install:
+  - pip install .
+  - pip install -r serveradmin/requirements.txt
+before_script:
+  - psql --version
+  - psql -U postgres -c 'SELECT version();'
+  - psql -c "create user serveradmin with password 'serveradmin';" -U postgres
+  - psql -c 'alter user serveradmin superuser;' -U postgres
+  - psql -c 'create database serveradmin;' -U postgres
+  - cp serveradmin/settings.example.py serveradmin/settings.py
+  - ./manage.py migrate
+script:
+  # This will use pythons standard unit test discovery feature.
+  # Name your tests like test*.py
+  - python3 -Wall ./manage.py test
```

Comment:
Have you check if `manage.py test` is not already enables `-Wall`?

Reply:
Yes, I have.